digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -473,6 +472,7 @@ object ProjectDAO {
   }
 
   def fetchReservedRoadParts(projectId: Long, filterNotStatus: Seq[Int] = Seq.empty[Int]): Seq[ReservedRoadPart] = {
+    // TODO: Check this filter: is it required to filter all road parts where ANY of the project links status is in filterNotStatus?
```

Comment:
the *filterNotStatus* seems to be not be passing through methods.  The filter could probably be removed.